test/cases/ostree.sh

```diff
@@ -686,6 +686,10 @@ ansible_private_key_file=${SSH_KEY}
 ansible_ssh_common_args="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
 EOF
 
+# Fix ansible error https://github.com/osbuild/osbuild-composer/issues/3309
+greenprint "fix stdio file non-blocking issue"
+sudo python3 /usr/libexec/osbuild-composer-test/ansible-blocking-io.py
```

Comment:
nitpick: You can drop `python3` once you add shebang to the script and make it executable. This applies to all files that include this.

Reply:
Done.